### PR TITLE
Modifying migrations play to work with edxapp

### PR DESCRIPTION
**Pull request by macdiesel**

### CODE REVIEW

**Comment by macdiesel** [comment]
@edx/pipeline-team PTAL.

File: playbooks/continuous_delivery/run_migrations.yml [hunk 1]
```diff
@@ -45,10 +49,28 @@
...
     shell: '{{ COMMAND_PREFIX }} run_migrations "{{ temp_output_dir.stdout }}/{{ unapplied_migrations_output }}" --output_file "{{ temp_output_dir.stdout }}/{{ migration_output }}"'
     become_user: "{{ APPLICATION_USER }}"
+    when: APPLICATION_NAME != "edxapp"
+
+  - name: migrate to apply any edxapp unapplied migrations
+    shell: '{{ COMMAND_PREFIX }} {{ SUB_APPLICATION_NAME }} run_migrations "{{ temp_output_dir.stdout }}/{{ unapplied_migrations_output }}" --database "{{ item }}" --settings "{{ EDX_PLATFORM_SETTINGS }}" --output_file "{{ temp_output_dir.stdout }}/{{ migration_output }}"'
+    become_user: "{{ APPLICATION_USER }}"
+    when: APPLICATION_NAME == "edxapp"
+    with_items:
+      - default
```
Comment (feanil) [inline playbooks/continuous_delivery/run_migrations.yml]: I'm not super excited about this being yet another place we have to remember to update when we add/change the number of databases in the edxapp.  Can we pull this out of the edxapp vars file?

Comment (feanil) [inline playbooks/continuous_delivery/run_migrations.yml]: maybe this: https://github.com/edx/configuration/blob/7c27fa57ce4bf27d4a571562fe303ecc5006ccab/playbooks/roles/edxapp/defaults/main.yml#L699

**Comment by feanil** [comment]
:+1:

**Comment by feanil** [comment]
Looks like this is failing in the loadtest cd job?

**Comment by macdiesel** [comment]
@edx/pipeline-team Please review. Database name variables are now picked out of the edxapp play.

**Comment by feanil** [comment]
:+1:

**Pull request merged**